.github/workflows/pull-test.yml

```
@@ -1,41 +1,98 @@
       - master
+  push:
+    branches:
+      - main
+      - master
+
 jobs:
-  build:
-    runs-on: ubuntu-20.04
+  pr:
+    # Pull requests: fast feedback on latest Python only
+    build:
```

actionlint (1.7.10)

unexpected key "build" for "job" section. expected one of "concurrency", "container", "continue-on-error", "defaults", "env", "environment", "if", "name", "needs", "outputs", "permissions", "runs-on", "secrets", "services", "snapshot", "steps", "strategy", "timeout-minutes", "uses", "with"

(syntax-check)